src/b3d/camera.py

```diff
@@ -138,26 +138,30 @@ def screen_from_camera(xyz: CameraCoordinates, intrinsics) -> ScreenCoordinates:
     Returns:
         (...,2) array of screen coordinates.
     """
-    # TODO: check this
-    xyz = jnp.clip(
-        xyz,
-        jnp.array([-jnp.inf, -jnp.inf, intrinsics.near]),
-        jnp.array([jnp.inf, jnp.inf, intrinsics.far]),
-    )
-    _, _, fx, fy, cx, cy, _, _ = intrinsics
+    _, _, fx, fy, cx, cy, near, far = intrinsics
     x, y, z = xyz[..., 0], xyz[..., 1], xyz[..., 2]
-    u = x * fx / z + cx
-    v = y * fy / z + cy
+    u_ = x * fx / z + cx
+    v_ = y * fy / z + cy
+
+    # TODO: What is the right way of doing this? Returning infs?
```

Comment:
I would cull to near and far instead of inf

Reply:
@mugamma  I am indifferent here. I wanted a value that indicates out-of-bounds explicitly, e.g. when a point is behind the camera, I don't want to lift it to the near plane.